website/src/components/TOS/PolicyContents.tsx

```diff
@@ -0,0 +1,380 @@
+export interface ChapterInfo {
+  id: number;
+  number: string;
+  title: string;
+  desc: string;
+  sections: SectionInfo[];
+}
+
+export interface SectionInfo {
+  id: number;
+  number: string;
+  title: string;
+  desc: string;
+}
+
+export const TermsData: ChapterInfo[] = [
```

Comment:
It feels a little weird to hide the contents here.  Can we define these in the actual pages? that would make it easy to look just at `website/src/pages/privacy-policy` and see the policy contents.  Alternatively, these could be embedded in a JSON file that we import.  

Also by moving these data declarations outside of here, you can move the interface declarations into the actual component definitions (since they're basically tied to the component definition).